### ISSUE TITLE
monitor the progress with DSK status.

### ISSUE THREAD

**Issue by jjhesk** [ericaltendorf/plotman#266]
I found out that iowait too long will lockup the progress of the actual plotting and they will have to do with DSK status. @altendky maybe we should take a closer look if that is possible to kill the process or suspend them until there are DSK capacity io available.

**Comment by BrunoCampana** [comment]
Hello @jjhesk , I'm currently have this issue of "DSK" with a lot of my plots. Did you manage to solve this in your plotting machine?

**Comment by jjhesk** [comment]
not really. I have just moving the plots around and try to avoid DSK bottleneck.

**Comment by altendky** [comment]
If your disk really can't handle that many plots, you could limit it to fewer plots.  Lots of choices depending on your setup.  Perhaps via the tmp overrides.
